sdk/ai/ai-projects/src/agents/vectorStores.ts

```diff
@@ -80,3 +81,47 @@ export async function deleteVectorStore(
   }
   return result.body; 
 }
+
+
+/**
+ * Creates a vector store and poll.
+ */
+export async function createVectorStoreAndPoll(
+  context: Client,
+  options?: CreateVectorStoreParameters,
+  sleepIntervalInMs?: number,
```

Comment:
These can be exposed as pollingOptions

Reply:
pollingOptions added in most recent commit